gms/impl/src/main/resources/index/data-process/settings.json

```diff
@@ -1,5 +1,6 @@
 {
   "index": {
+    "max_ngram_diff": 47,
```

Comment:
Is this required? What happens if we do not specify?

Reply:
Yes. If you don't then making the index fails and ES tells you to set the setting.